accounts-journey/src/main/kotlin/com/backbase/accounts_journey/data/usecase/AccountDetailUseCase.kt

```diff
@@ -1,10 +1,10 @@
 package com.backbase.accounts_journey.data.usecase
 
-import com.backbase.accounts_journey.domain.model.account_detail.AccountDetail
+import com.backbase.android.client.gen2.arrangementclient2.model.AccountArrangementItem
 import dev.drewhamilton.poko.Poko
 
 interface AccountDetailUseCase {
```

Comment:
```suggestion
fun interface GetAccountDetailUseCase {
```

Reply:
I recommend this change to enforce the SAM nature of this use-case and name it accordingly.